src/azure-cli/azure/cli/command_modules/storage/_help.py

```diff
@@ -221,7 +225,8 @@
   - name: Get a connection string for a storage account.
     text: az storage account show-connection-string -g MyResourceGroup -n MyStorageAccount
   - name: Get the connection string for a storage account. (autogenerated)
-    text: az storage account show-connection-string --name MyStorageAccount --resource-group MyResourceGroup --subscription MySubscription
+    text: |
+        az storage account show-connection-string --name MyStorageAccount --resource-group MyResourceGroup --subscription MySubscription
```

Comment:
At first, this example name is the same with above one.
Secondly, --subscription is a global argument not this command specific one. Looks that above one is enough.

Reply:
Yeah, the same situation met in network command examples. 
Should reply to Aladdin team.